src/declarativewebutils.cpp

```diff
@@ -411,7 +378,7 @@ void DeclarativeWebUtils::handleObserve(const QString message, const QVariant da
 
                     // load opensearch descriptions
                     loadsearch.insert(QString("msg"), QVariant(QString("loadxml")));
-                    loadsearch.insert(QString("uri"), QVariant(QString("chrome://embedlite/content/") +
+                    loadsearch.insert(QString("uri"), QVariant(QString("file://") + OpenSearchConfigs::userOpenSearchPath() +
```

Comment:
This won't work. Try to delete /home/nemo/.mozilla and restart the browser: the search descriptors won't be loaded.
First `configs` will be initialized to the content of `chrome://embedlite/content/` and `registeredSearches` to an empty list. Then in this line we'll try to loaded a pre-installed search descriptor from the user folder -> fail.